### ISSUE TITLE
[Timeline] Weird timeline behaviour with many logpoint markers

### ISSUE THREAD

**Issue by hbenl** [RecordReplay/devtools#339]
When setting a logpoint that generates many messages, the timeline starts acting weird, with markers appearing and disappearing again.

https://replay.io/view?id=ab315f0a-d05a-4151-9eb1-18fda1a5c261

**Comment by jasonLaster** [comment]
Could you describe what you're seeing? The recording is not very clear.

I also saw this when replaying :)
<img width="929" alt="Screen Shot 2020-08-15 at 8 19 24 AM" src="https://user-images.githubusercontent.com/254562/90315397-1976c780-ded0-11ea-9aef-d1fd9706306f.png">

**Comment by hbenl** [comment]
Try to click somewhere in the middle of the timeline, then step forward/backward using the buttons next to the play button.
You'll see that after adding the logpoint, after a while markers are added to the timeline (this is the expected behavior) but then some of them disappear and sometimes reappear again.
In another example with a logpoint that was hit even more often, I saw this appearing and disappearing of markers going on for about half a minute before it eventually settled on a set of markers that was obviously incomplete because when I clicked the messages in the console I often ended up in a place on the timeline that had no marker anywhere near it.
